### PR TITLE
COR-363 add stubs form cases & recipients

### PR DESCRIPTION
**Pull request by ludydoo**

### CODE REVIEW

File: pkg/api/types/recipientdef.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+	// Extends represents the Parent of the Recipient. Sometimes, we need to have multiple "profiles"
+	// for case recipients. For example, Uganda Beneficiary Individual, Kenya Beneficiary Individual,
+	// Colombia BeneficiaryIndividual are 3 different recipients that "extend" the same "Global Beneficiary Individual"
+	// In this case, Uganda Beneficiary Individual would have Global Beneficiary Individual as Parent
+	//
+	// Also, in some cases, the parent might be a custom FormDefinition that is not a RecipientDef.
+	// If the user has already defined a custom free-form, eg. Individual, and wants to add Case Management
+	// to this currently existing form, then the Extends property would reference the Individual FormDefinition.
+	Extends *FormRef `json:"extends,omitempty"`
```
Comment (internetti) [inline pkg/api/types/recipientdef.go:16]: what do you think about naming this "Parent"? I think that way it's clearer what this is

Comment (ludydoo) [reply pkg/api/types/recipientdef.go]: Sounds good :) Just changed it

File: pkg/api/types/recipientdef.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+	// Parent represents the Parent of the Recipient. Sometimes, we need to have multiple "profiles"
+	// for case recipients. For example, Uganda Beneficiary Individual, Kenya Beneficiary Individual,
+	// Colombia BeneficiaryIndividual are 3 different recipients that "extend" the same "Global Beneficiary Individual"
+	// In this case, Uganda Beneficiary Individual would have Global Beneficiary Individual as Parent
+	//
+	// Also, in some cases, the parent might be a custom FormDefinition that is not a RecipientDef.
+	// If the user has already defined a custom free-form, eg. Individual, and wants to add Case Management
+	// to this currently existing form, then the Extends property would reference the Individual FormDefinition.
+	Parent *FormRef `json:"extends,omitempty"`
```
Comment (neb42) [inline pkg/api/types/recipientdef.go:16]: Why are you renaming this to `extends` instead of `parent`?

**Pull request merged**